Documentation/teaching/labs/interrupts.rst

```diff
@@ -1024,8 +1024,8 @@ minor are defined as ``KBD_MAJOR`` and ``KBD_MINOR``:
    cat /dev/kbd
 
 
-Reset the buffer
-................
+5. Reset the buffer
+-----------------------------
```

Comment:
Hi,

The number of dots must be exactly the same as the number of characters above.